src/pages/Settings/Settings.js

```diff
@@ -1,146 +1,182 @@
 import React, { useState } from 'react'
-import useForm from 'react-hook-form'
-import * as yup from 'yup'
+import { useQuery, useMutation } from '@apollo/react-hooks'
+import { isEmpty, capitalize } from 'lodash/fp'
+import './Settings.module.css'
+import { sliceHash as presentHash } from 'utils'
+import HashAvatar from 'components/HashAvatar'
+import Modal from 'components/Modal'
 import PrimaryLayout from 'components/layout/PrimaryLayout'
 import Button from 'components/Button'
-import Input from 'components/Input'
+import HposSettingsQuery from 'graphql/HposSettingsQuery.gql'
+import HposStatusQuery from 'graphql/HposStatusQuery.gql'
+import HposUpdateVersionMutation from 'graphql/HposUpdateVersionMutation.gql'
 
-import './Settings.module.css'
+export const createLabelfromSnakeCase = string => string.replace(/([A-Z])/g, ' $1').split(' ').map(word => capitalize(word)).join(' ')
```

Comment:
This is clever, but I think the code would be more self documenting if this were just a dictionary lookup
```
getLabelFromPortName = portname => ({
  primaryPort: 'Primary Port'.
  // ...etc
}[portname])
```

Does that make sense? Maybe I'm missing something.

Reply:
This works too and upon second reflection, I believe would also be better control over the output. 
 I'll switch to this approach.